mod/gccollab_theme/views/default/page/elements/user_menu.php

```diff
@@ -99,14 +99,7 @@
                 ));
             ?>
         </li>
-        <li class="elgg-menu-item-colleagues">
-            <?php
-                // colleagues
-                echo '<a href="/friends/'.$username.'" title="'.elgg_echo("userMenu:colleagues").'" class="elgg-menu-content">
-                    <span class="fa fa-users fa-lg"><span class="wb-invisible">'.elgg_echo("userMenu:colleagues").'</span></span>'.$friend_badge.'
-                </a>';
-            ?>
-        </li>
+        <!-- CL 20221213 - Removed Friends Nav button re: https://github.com/gctools-outilsgc/gcconnex/issues/2474 -->
```

Comment:
Probably don't need a comment, since the git history covers that with a good commit message